G-licious/Controllers/StationsViewController.m

```diff
@@ -1,405 +0,0 @@
-//
```

Comment:
You can restore each by using `git checkout 644a32ce6ba2b5746a070e4165e9f2e544b23235 <filename>`. I'll leave it up to you, but we'll have to do it now or later, and I'm always a proponent of frontloading work; especially in case somebody forgets we have these files in our version history.